test/benchmarks/src/main/scala/scala/reflect/NamesBenchmark.scala

```diff
@@ -111,18 +133,23 @@ class NamesBenchmark extends AbstractNamesBenchmark {
     var i = 0
     val res = new Array[AnyRef](arraySize)
     while (i < arraySize) {
-      val c= namesChars(i)
-      res(i) = nameTable.find(c,0,c.length)
+      val c = namesChars(i)
+      res(i) = nameTable.find(c, 0, c.length)
```

Comment:
I would `bh.consume` each value from the loop rather than writing them into an array and consuming that.

Reply:
I used to have that. The problem is that the weak values start being cleaned as the test progresses, so we are then not testing the building of a name table